tesler-core/src/main/java/io/tesler/core/metahotreload/service/ScreenAndNavigationGroupAndNavigationViewUtil.java

```diff
@@ -0,0 +1,120 @@
+/*-
+ * #%L
+ * IO Tesler - Core
+ * %%
+ * Copyright (C) 2018 - 2019 Tesler Contributors
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+package io.tesler.core.metahotreload.service;
+
+import static io.tesler.core.metahotreload.util.JsonUtils.serializeOrElseNull;
+import static java.util.Optional.ofNullable;
+
+import io.tesler.core.metahotreload.dto.ScreenSourceDto;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.tesler.api.data.dictionary.CoreDictionaries;
+import io.tesler.model.ui.entity.Screen;
+import io.tesler.model.ui.navigation.NavigationGroup;
+import io.tesler.model.ui.navigation.NavigationView;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+import javax.annotation.Nullable;
+import javax.persistence.EntityManager;
+import lombok.NonNull;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class ScreenAndNavigationGroupAndNavigationViewUtil {
+
+	static void process(
+			@NonNull List<ScreenSourceDto> screenDtos,
+			@NonNull EntityManager entityManager,
```

Comment:
We use beans here. Why we don't declare it as a bean and don't autowire entityManager and objMapper?
Same for similar classes.

Reply:
fixed